org.elysium.parent/org.elysium/src/org/elysium/importuri/LilyPondImportUriResolver.java

```diff
@@ -28,11 +29,23 @@
  */
 public class LilyPondImportUriResolver extends ImportUriResolver {
 
+	private static final String UNRESOLVED_URI_PREFIX = "really_did_not_find_include-";
```

Comment:
Please introduce a type (e.g. a `@Data` class) to represent whether the URI is "really resolved" instead of String. (Or maybe reuse LilyPondImportUri?)

Reply:
LilyPondImportUri should not be reused as it is currently used heavily by the refactoring code which is not changed yet.